Matchismo/Matchismo/Model/SetCardDeck.mm

```diff
@@ -13,31 +13,27 @@
 
 @implementation SetCardDeck
 
-- (NSArray *)defaultShapes {
-  return @[@"●", @"■", @"▲"];
-}
 
 - (NSArray *)defaultColors {
   return @[UIColor.redColor, UIColor.greenColor, UIColor.purpleColor];
 }
 
 - (instancetype)init {
-  return [self initWithShapes:[self defaultShapes] colors:[self defaultColors]];
+  return [self initWithColors:[self defaultColors]];
 }
 
-- (instancetype)initWithShapes:(NSArray *)shapes colors:(NSArray *)colors {
+- (instancetype)initWithColors:(NSArray *)colors {
   if (self = [super init]) {
-    if (shapes.count != 3) {
-      shapes = [self defaultShapes];
-    }
     if (colors.count != 3) {
       colors = [self defaultColors];
     }
+    
     for (NSUInteger noOfShapes = 1; noOfShapes < 4; noOfShapes++) {
-      for (NSString *shape in shapes) {
+      for (int shapeVal = 0; shapeVal < setCardShapeCount; shapeVal++) {
         for (UIColor *color in colors) {
           for (int shadingVal = 0; shadingVal < setCardShadingCount; shadingVal++) {
             SetCardShading shading = static_cast<SetCardShading>(shadingVal);
+            SetCardShape shape = static_cast<SetCardShape>(shapeVal);
```

Comment:
You don't need to use `static_cast` just to set an integer value to an NS_ENUM. 
Obj-c is much more relaxed than Swift here - you can just assign the values directly.